db.json

```diff
@@ -1,5 +1,35 @@
 {
   "workstreams": [
+    {
+      "id": "browser-testing-api",
```

Comment:
I have deliberate chosen an unwieldy name here that we need to change, but I'm not sure what to name this. A single word would be great, but I'm not sure if just "test" and "testing" is great. Thoughts?

Reply:
Well the precednt is that "api" is unneeded, so `browser-test[ing]` is already shorter. I also don't really mind `testtutils`.

Reply:
We also don't have any hyphens in existing shortnames. Between "browsertest[ing]" and "testutils", do you have a preference?

Reply:
I don't really. `fetch` and `console` are named after the primary interface they provide, but `dom` and `html` aren't. Ptobably the precent of those two would suggest you actually want an acronym for multi-word cases and so it would be the "Testing API (for) Browsers" i.e. `tab` (not a serious suggestion ;) ).

Reply:
If we go with "testutils" would the name just be "Test Utils"? Seems okay with me. Avoiding hyphens sounds nice.